src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -307,9 +293,11 @@ public static void ThrowIfNotOk(sqlite3 handle, Result result)
         public void Throw(Result result)
             => Throw(_handle, result);
 
-        public static void Throw(sqlite3 handle, Result result)
-            => throw new SqlException(result,
-                raw.sqlite3_errmsg(handle) + "\r\n" +
-                raw.sqlite3_errstr(raw.sqlite3_extended_errcode(handle)));
+        public static void Throw(SafeSqliteHandle handle, Result result)
+        {
+            throw new SqlException(result,
+                NativeMethods.sqlite3_errmsg(handle) + "\r\n" +
```

Comment:
"\r\n" even on Linux or macOS?  Environment.NewLine?

Reply:
Will fix in master